velox/core/PlanNode.cpp

```diff
@@ -126,11 +128,13 @@ void AggregationNode::addDetails(std::stringstream& stream) const {
     stream << "] ";
   }
 
+  bool isNotFirstName = false;
   for (auto i = 0; i < aggregateNames_.size(); ++i) {
-    if (i > 0) {
+    if (isNotFirstName) {
```

Comment:
What's the motivation for this change?

Reply:
I feel this has better code readability. But if you find it to be verbose, I can revert it. 
I have seen a similar pattern here. https://github.com/facebookincubator/velox/blob/main/velox/vector/SelectivityVector.h#L300

Reply:
I feel (i > 0) is easier to read. In the example you shared "i" doesn't work.

Reply:
Done.